### PR TITLE
Feature recordlines n

### PR DESCRIPTION
**Pull request by tjerman**

### CODE REVIEW

File: src/compose/types/page-block/record-list.ts [hunk 1]
```diff
@@ -22,6 +23,12 @@ interface Options {
   allowExport: boolean;
   perPage: number;
 
+  // Record-lines
+  inlineEditor: boolean;
```
Comment (darh) [inline src/compose/types/page-block/record-list.ts]: this is always implicit when editFields are there or any of draggable, positionField, parentField is set, correct?
anyhow, if it really needs to stay, `editable` might be better (so, same as `draggable`)

File: src/compose/types/page-block/record-list.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ interface Options {
   prefilter: string;
   presort: string;
   fields: unknown[];
+  editFields?: unknown[];
```
Comment (darh) [inline src/compose/types/page-block/record-list.ts]: move this down under // record-lines

File: src/compose/types/page-block/record-list.ts [hunk 3]
```diff
@@ -22,6 +23,12 @@ interface Options {
   allowExport: boolean;
   perPage: number;
 
+  // Record-lines
+  inlineEditor: boolean;
+  draggable?: boolean;
+  positionField?: string;
+  parentField?: string;
```
Comment (darh) [inline src/compose/types/page-block/record-list.ts]: not loving this "parent"... "refField"? this way we use the same terminology...

**Review by darh — CHANGES_REQUESTED** [review]
minor stuff

**Comment by tjerman** [comment]
Merged

**Pull request closed without merging**